charmcraft.yaml

```diff
@@ -1,11 +1,42 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
-
+name: snmp-exporter
 type: charm
-bases:
-  - build-on:
-    - name: ubuntu
-      channel: "22.04"
-    run-on:
-    - name: ubuntu
-      channel: "22.04"
+summary: Prometheus compatible SNMP exporter.
+description: |
+  Exporter that exposes information gathered from SNMP for use by the a Prometheus compatible monitoring system.
+
+links:
+  documentation: https://discourse.charmhub.io/t/snmp-exporter-docs-index/12050
+  website: https://charmhub.io/snmp-exporter
+  source: https://github.com/canonical/snmp-exporter-operator
+  issues: https://github.com/canonical/snmp-exporter-operator/issues
+
+
+platforms:
+  ubuntu@22.04:amd64:
+  ubuntu@24.04:amd64:
```

Comment:
This is a vm charm so I guess it's fine to keep those here?

Reply:
Okay :)